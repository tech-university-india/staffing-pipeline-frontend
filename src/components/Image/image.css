img {
  width: 100%;
  height: 100%;
  object-fit: cover;
}

.circle-image {
  border-radius: 50%;
}
