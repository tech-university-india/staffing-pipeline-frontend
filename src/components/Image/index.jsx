import React from 'react';
import './image.css';
import PropTypes from 'prop-types';

export default function Image({ imageUrl, altText, isACircle }) {
  return <img src={imageUrl} alt={altText} className={`${isACircle ? 'circle-image' : ''}`} />;
}

Image.propTypes = {
  imageUrl: PropTypes.string.isRequired,
  altText: PropTypes.string.isRequired,
  isACircle: PropTypes.bool,
};
