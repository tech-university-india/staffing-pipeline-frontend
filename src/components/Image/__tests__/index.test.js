import React from 'react';
import { render } from '@testing-library/react';
import Image from '..';

describe('Image Component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Image />);
    expect(asFragment()).toMatchSnapshot();
  });
});
