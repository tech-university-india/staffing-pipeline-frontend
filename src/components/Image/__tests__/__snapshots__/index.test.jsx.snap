// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Component should render correctly 1`] = `
<DocumentFragment>
  <img
    class=""
  />
</DocumentFragment>
`;
